bigquery/indexer.py

```diff
@@ -317,19 +317,22 @@ def create_samples_json_export_file(es, index_name, deploy_project_id):
             })
 
     client = storage.Client(project=deploy_project_id)
+    user = client.get_service_account_email(
+        project=deploy_project_id).split('@')[0]
     # Don't put in project_id-export because that bucket has TTL= 1 day.
     bucket_name = '%s-export-samples' % deploy_project_id
     bucket = client.lookup_bucket(bucket_name)
     if not bucket:
         bucket = client.create_bucket(bucket_name)
-    blob = bucket.blob('samples')
+    samples_file_name = '%s-%s-samples' % (index_name, user)
```

Comment:
Can you also send corresponding PR for data-explorer repo?

For the code to determine username, I guess we'll have to copy-paste code with a comment "// Keep in sync with ...".

Note this samples file will go away once Saturn can run SQL queries.